models/userDao.js

```diff
@@ -63,7 +63,22 @@ const createKakaoUser = async (
   );
 };
 
+const getUserInfo = async (kakaoId) => {
+  const userId = await appDataSource.query(
+    `
+    SELECT  
+    id as userId
+    FROM users
+    WHERE social_id =?
+      `,
+    [kakaoId]
+  );
+
+  return userId;
+};
```

Comment:
```suggestion
const getUserIdBySocialId = async (socialId) => {
  return appDataSource.query(
    `
    SELECT  
    id as userId
    FROM users
    WHERE social_id =?
      `,
    [kakaoId]
  );
};
```
1. 변수 선언 없이 곧바로 return 해도 되는 코드입니다.
2. 함수의 결과값이나 내부 동작을 고려해보면 다음과 같이 함수 naming 수정과 함께 parameter를 수정해야 합니다.